### ISSUE TITLE
Game Crashes after the advance SDK screen

### ISSUE THREAD

**Issue by jamesgamer3** [Hary309/hry-sic#10]
[game.log.txt](https://github.com/Hary309/hry-sic/files/8401185/game.log.txt)
[game.crash.txt](https://github.com/Hary309/hry-sic/files/8401186/game.crash.txt)
[hry_core.log](https://github.com/Hary309/hry-sic/files/8401188/hry_core.log)

**Comment by Hary309** [comment]
Looks like teamspeak overlay is fighting with my plugin. Can't do much about that right now, sorry.

**Comment by jamesgamer3** [comment]
Huh? I don't have Teamspeak?

**Comment by Hary309** [comment]
I mean the Overwolf, it is a teamspeak creation.

**Comment by jamesgamer3** [comment]
Ah, but you mean the app its self cause a friend of mine can use over wolf with your plugin fine but anyways I'll get rid of it an see if it fixes the problem

**Comment by Hary309** [comment]
Oh, that's interesting, I wish I had time to investigate the issue 😞

**Comment by jamesgamer3** [comment]
Sadly no it still crashes, and I fully uninstalled over wolf and the plugin still cashes I sent the crash logs and stuff just to be safe
[game.crash.txt](https://github.com/Hary309/hry-sic/files/8885565/game.crash.txt)
[game.log.txt](https://github.com/Hary309/hry-sic/files/8885566/game.log.txt)
[hry_core.log](https://github.com/Hary309/hry-sic/files/8885567/hry_core.log)

**Comment by jamesgamer3** [comment]
Oh, quick update, I got it to work Apparently when I uninstalled Overwolf the app I used with it trucky left this behind and when I deleted it your plugin works fine, so I feel like something with this might be messing with your plugin causing it to crash 
[Image](https://imgur.com/a/CNUh8mQ)

**Comment by Hary309** [comment]
Thank you for the information, I will try to do some investigation.

**Comment by rcj2** [comment]
My game does the same crashes after the SDK screen

**Comment by rcj2** [comment]
[game.log.txt](https://github.com/Hary309/hry-sic/files/10041871/game.log.txt)
[game.crash.txt](https://github.com/Hary309/hry-sic/files/10041875/game.crash.txt)

**Comment by Hary309** [comment]
Looks like mod isn't compatible with the newest version of the game. I will take a look later.

**Comment by Hary309** [comment]
I looked closely at this and this seems really weird, are you sure you are using the newest version?

**Comment by rcj2** [comment]
Yep download it a couple of times to make sure it wasn't just me

**Comment by Hary309** [comment]
Can you try running it on ETS2? (if you have)

**Comment by rcj2** [comment]
I'll have a try tomorrow for you

**Comment by rcj2** [comment]
Works on ets2 but not on ats

**Comment by Hary309** [comment]
Great, at least this...
Could you send file hry_core.log from `<Documents>/hry_core/ats`?

**Comment by rcj2** [comment]
I do not have that?

**Comment by rcj2** [comment]
![American Truck Simulator 22_11_2022 22_59_09](https://user-images.githubusercontent.com/118542421/203438370-17e7977c-c8f1-4b4f-9e53-692640abc3c8.png)

**Comment by Hary309** [comment]
It is different directory, outside of this.

**Comment by rcj2** [comment]
[hry_core.log](https://github.com/Hary309/hry-sic/files/10077164/hry_core.log)

**Comment by Hary309** [comment]
Looks like you have outdated hry-sic.dll. Please remove `plugins` from `<path_to_game>\bin\win_x64`, and then install the plugin again.

**Comment by rcj2** [comment]
I have done that like 3 times past couple days I will have another try later

**Comment by rcj2** [comment]
Nope downloaded the new one off your website and still does the same

**Comment by ldee247** [comment]
Hi, help needed I have been having series of Ets2 crashing to desktop or sometimes forced to reboot pc . This problem occurs soon after loading Ets2.
Sometimes the screen freezes randomly after or before clicking continue to game or drive icon. I have been getting Kernellbase.dll and d3d11.dll 
![2024-01-30_18-46-57](https://github.com/Hary309/hry-sic/assets/10225541/02b37dea-d3ff-4c5d-b6b4-92382cdd70cf)
![2024-01-29_22-21-22](https://github.com/Hary309/hry-sic/assets/10225541/c787e390-09a5-4143-b0cf-47f8b1be2241)


My Ets2 was running all ok before installing the camera plugin.

After checking today through the system security and maintenance only realized the plugin has been the source of crashes.

 
![2024-02-01_15-42-07](https://github.com/Hary309/hry-sic/assets/10225541/df124432-7b78-41e6-a2be-163dc420b90d)
![2024-02-01_15-44-56](https://github.com/Hary309/hry-sic/assets/10225541/6af48d1c-89df-47b3-837d-4980b34a8475)
 
Herewith Crash logs:

[game.crash.txt](https://github.com/Hary309/hry-sic/files/14127591/game.crash.txt)
[game.log.txt](https://github.com/Hary309/hry-sic/files/14127595/game.log.txt)
[hry_core.log](https://github.com/Hary309/hry-sic/files/14127599/hry_core.log)

I hope this can be resolved as its a very great helpful camera plugin.

How do I uninstall for the meantime and try to install if there has been a new fix that was rectified.

Thanks in advance
